File: httpStatusCategorizer.js
function categorizeStatusCode(code) {
    if (code >= 200 && code <= 299) {
        return "Success";
    } else if (code >= 300 && code <= 399) {
        return "Redirection";
    } else if (code >= 400 && code <= 499) {
        return "Client Error";
    } else if (code >= 500 && code <= 599) {
        return "Server Error";
    } else {
        return "Invalid";
    }
}

// Main execution
const args = process.argv.slice(2);
const code = Number(args[0]);

if (Number.isNaN(code)) {
    console.log("Output: Invalid");
} else {
    console.log(`Output: ${categorizeStatusCode(code)}`);
}

File: http_status_categorizer.py
def categorize_status_code(code):
    if 200 <= code <= 299:
        return "Success"
    elif 300 <= code <= 399:
        return "Redirection"
    elif 400 <= code <= 499:
        return "Client Error"
    elif 500 <= code <= 599:
        return "Server Error"
    else:
        return "Invalid"

if __name__ == "__main__":
    try:
        code = int(input("Input: "))
        print(f"Output: {categorize_status_code(code)}")
    except ValueError:
        print("Output: Invalid")
